integreat_cms/cms/templates/_base.html

```diff
@@ -187,6 +187,14 @@
                         </a>
                     </div>
                 {% endif %}
+                {% if perms.cms.manage_translations %}
```

Comment:
This is quite unfortunate, but the screenshot is a bit outdated. In the meantime we decided to move the contact below POI/locations. Could you please move this section below POI? Thank you so much & I'm sorry for the confusion!

Reply:
Thank you for the info.
I have updated `_base.html` accordingly.